aries-site/src/pages/design-tokens/component-states.mdx

```diff
@@ -0,0 +1,70 @@
+
+Application states that have use cases within the design system are represented in the table below.
+
+## Supported states
+
+In this table, “interaction” states are represented as column headings and “application” states are the rows. Checked cells represent valid combinations.
+
+If two application states exist at once, the component should be styled based on the top-most row in the table. For example, if a component is both “readonly” and "selected", it should be styled as "readonly".
+
+<Notification
+  status="info"
+  message="Active and focus state does not currently have design tokens, but they are included in this model to demonstrate how the system would extend if styles for this state are added. For focus state, use the global focusIndicator tokens."
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+/>
+
+|               |                  rest                   |                  hover                  |                  focus                  |                 active                  | Notes                     |
+| ------------- | :-------------------------------------: | :-------------------------------------: | :-------------------------------------: | :-------------------------------------: | ------------------------- |
+| none          | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | --                        |
+| error         | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | --                        |
+| disabled      | <span><Checkmark size="small" /></span> |                   --                    |                   --                    |                   --                    | --                        |
+| readonly      | <span><Checkmark size="small" /></span> |                   --                    | <span><Checkmark size="small" /></span> |                   --                    | --                        |
+| selected      | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | --                        |
+| indeterminate | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | Only applies to checkbox. |
+| pinned        | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | <span><Checkmark size="small" /></span> | --                        |
+
+## Component-specific tokens
+
+For each component with [component-specific tokens](/design-tokens/how-to-read-design-tokens#component), the applicable state combinations are provided as design tokens.
```

Comment:
Also, the table on the "how to read tokens" refers to "enabled" instead of "rest".

Reply:
Can you show me where you're seeing this still? I believe I fixed references to "enabled" in a PR on master. should be resolved already

Reply:
looks to be resolved.